common/models.py

```diff
@@ -144,6 +150,7 @@ class Meta:
 class TwoColumnBlock(blocks.StructBlock):
     
     background = blocks.ChoiceBlock(choices=COLOUR_CHOICES,default="white")
+    left_column_size = blocks.ChoiceBlock(choices=COLUMN_CHOICES,default="6")
     left_column = blocks.StreamBlock([
```

Comment:
Do we want editors to specify all widths of columns? ie left is half and right is a third?